mlflow/pipelines/steps/train.py

```diff
@@ -605,84 +683,20 @@ def render_schema(inputs, title):
-                        "'enabled' key whose value is either true or false.",
-                        error_code=INVALID_PARAMETER_VALUE,
-                    )
-
-                if step_config["tuning_enabled"]:
-                    if "sample_fraction" in step_config["tuning"]:
-                        sample_fraction = float(step_config["tuning"]["sample_fraction"])
-                        if sample_fraction > 0 and sample_fraction <= 1.0:
-                            step_config["sample_fraction"] = sample_fraction
-                        else:
-                            raise MlflowException(
-                                "The tuning 'sample_fraction' configuration in the train step "
-                                "must be between 0 and 1.",
-                                error_code=INVALID_PARAMETER_VALUE,
-                            )
-                    else:
-                        step_config["sample_fraction"] = 1.0
-
-                    if "algorithm" not in step_config["tuning"]:
-                        step_config["tuning"]["algorithm"] = "hyperopt.rand.suggest"
-
-                    if "parallelism" not in step_config["tuning"]:
-                        step_config["tuning"]["parallelism"] = 1
-
-                    if "max_trials" not in step_config["tuning"]:
-                        raise MlflowException(
-                            "The 'max_trials' configuration in the train step must be provided.",
-                            error_code=INVALID_PARAMETER_VALUE,
-                        )
-
-                    if "parameters" not in step_config["tuning"]:
-                        raise MlflowException(
-                            "The 'parameters' configuration in the train step must be provided "
-                            " when tuning is enabled.",
-                            error_code=INVALID_PARAMETER_VALUE,
-                        )
-
-            else:
-                step_config["tuning_enabled"] = False
-
-            if "estimator_params" not in step_config:
-                step_config["estimator_params"] = {}
-
-            step_config.update(
-                get_pipeline_tracking_config(
-                    pipeline_root_path=pipeline_root,
-                    pipeline_config=pipeline_config,
-                ).to_dict()
-            )
-        except KeyError:
-            raise MlflowException(
-                "Config for train step is not found.", error_code=INVALID_PARAMETER_VALUE
-            )
+        step_config = {}
+        if pipeline_config.get("steps", {}).get("train", {}) is not None:
+            step_config.update(pipeline_config.get("steps", {}).get("train", {}))
+        step_config["metrics"] = pipeline_config.get("metrics")
+        step_config["template_name"] = pipeline_config.get("template")
+        step_config["profile"] = pipeline_config.get("profile")
+        step_config["run_args"] = pipeline_config.get("run_args")
```

Comment:
Can we remove `run_args`, since it breaks caching? I.e. if a user runs `mlflow pipelines run --step train` and then `mlflow pipelines run --step evaluate`, `train` will be retriggered even though nothing about the train step has changed since `run_args` is different.